frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
Not sure to see the point of adding this comment as it is already done like this in a lot of places.